game/scripts/vscripts/units/twin.lua

```diff
@@ -0,0 +1,29 @@
+LinkLuaModifier( "modifier_boss_phase_controller", "modifiers/modifier_boss_phase_controller", LUA_MODIFIER_MOTION_NONE )
+
+function Spawn (entityKeyValues)
+  thisEntity:FindAbilityByName("boss_twin_twin_empathy")
+  thisEntity:FindAbilityByName("boss_twin_spawn_twin")
+
+  thisEntity:SetContextThink( "TwinThink", partial(TwinThink, thisEntity) , 1)
+  print("Starting AI for " .. thisEntity:GetUnitName() .. " " .. thisEntity:GetEntityIndex())
+
+  ABILITY_spawn_twin = thisEntity:FindAbilityByName("boss_twin_spawn_twin")
+  ABILITY_empathy = thisEntity:FindAbilityByName("boss_twin_twin_empathy")
+
+  local phaseController = thisEntity:AddNewModifier(thisEntity, ABILITY_empathy, "modifier_boss_phase_controller", {})
+  phaseController:SetPhases({ 75, 50 })
+  phaseController:SetAbilities({
+    "boss_twin_twin_empathy"
+  })
+end
+
+function TwinThink()
+  ExecuteOrderFromTable({
```

Comment:
Instead of using an ability to do this, since we only do it once and kind of want strong control over it, lets do something like...
```lua
local twin = CreateUnitByName("npc_dota_boss_twin_dumb", self:GetAbsOrigin(), true, self, self:GetOwner(), self:GetTeam())
twin:AddNewModifier(self, ABILITY_empathy, "modifier_boss_twin_twin_empathy", {})
```

This makes it so there's only 1 instance of the modifier, and it'll have the empathy ability's KV and levels at run time.